server.js

```diff
@@ -1,5 +1,16 @@
-const app = require("./app");
```

Comment:
zakładam że przez pomyłkę to usunęłaś, ale plik app.js oraz listen jest potrzebny do obsługi routów.